utils/add-file/mount-file.batch.yaml

```diff
@@ -0,0 +1,33 @@
+name: add-file-at-result-path-using-mounting
+description: Add file to all path with a search result. In this example, we add an Apache license beside each README.md file.
+
+# Find all repositories that contain a README.md file.
+on:
+  - repositoriesMatchingQuery: file:README.md count:200
+
+# Besides each match, add a readme file. Note that repository.search_result_paths is a helper variable provided by the spec templating language.
```

Comment:
```suggestion
# Beside each match, add a LICENSE.md file. Note that repository.search_result_paths is a helper variable provided by the spec templating language.
```

Reply:
I think this is wrong in the original spec, too, but I can't leave a comment there since it's not part of the diff. 🤷‍♀️